website/docs/backends/types/swift.html.md

```diff
@@ -99,11 +101,11 @@ The following configuration options are supported:
    If omitted the `OS_KEY` environment variable is used.
 
  * `archive_container` - (Optional) The container to create to store archived copies
-   of the Terraform state file. If specified, Swift [object versioning](https://docs.openstack.org/developer/swift/overview_object_versioning.html) is enabled on the container created at `container`.
+   of the Terraform state file. If specified, Swift [object versioning](https://docs.openstack.org/developer/swift/latest/overview_object_versioning.html) is enabled on the container created at `container`.
 
  * `archive_path` - (Optional) DEPRECATED: Use `archive_container` instead.
    The path to store archived copied of `terraform.tfstate`. If specified,
-   Swift [object versioning](https://docs.openstack.org/developer/swift/overview_object_versioning.html) is enabled on the container created at `path`.
+   Swift [object versioning](https://docs.openstack.org/developer/swift/latest/overview_object_versioning.html) is enabled on the container created at `path`.
 
- * `expire_after` - (Optional) How long should the `terraform.tfstate` created at `path`
-   be retained for? Supported durations: `m` - Minutes, `h` - Hours, `d` - Days.
+ * `expire_after` - (Optional) How long should the `terraform.tfstate` created at `container`
```

Comment:
I'd suggest we want a warning box on this field too (since Terraform is inherently stateful) - and ditching the Statefile could cause issues - what do you think? e.g.

```
~> **NOTE:** Since Terraform is inherently stateful - we'd strongly recommend against auto-expiring Statefiles.
```

Reply:
Totally agree. I'll add that in and rebase since another change landed recently.

Reply:
Done. Please let me know if there's any other improvements that need to be made.

Reply:
thanks for making those changes. I'll ask @radeksimko just to give this a second glance for phrasing but this otherwise LGTM 👍